src/cloudant/__init__.py

```diff
@@ -62,6 +63,56 @@ def cloudant(user, passwd, **kwargs):
     cloudant_session.disconnect()
 
 @contextlib.contextmanager
+def cloudant_bluemix(service_name, **kwargs):
```

Comment:
Agreed, this would be better. Addressed in squash commit 05acaee0b270a503b264ed31f70908e805e65017.